src/pages/cart/cart.tsx

```diff
@@ -0,0 +1,62 @@
+import { useContext } from "react";
+import { useNavigate } from "react-router-dom";
+import CartRow from "../../components/cart-row/cart-row";
+import EmptyCart from "../../components/empty-cart/empty-cart";
+import { CartContextObject } from "../../entities/cart-context-object";
+import { CartContext } from "../../providers/cart-provider";
+
+const Cart = () => {
+  const { cart, setCart } = useContext<CartContextObject>(CartContext);
+  const navigate = useNavigate();
+
+  return (
+    <>
+      {cart.length === 0 ? (
+        <EmptyCart />
+      ) : (
+        <div className="cart">
+          <h1>Cart</h1>
+          <div className="cart-items">
+            <div className="cart-headings">
+              <span>Image</span>
+              <span>Name</span>
+              <span>Category</span>
+              <span>Quantity</span>
+              <span>Price</span>
+              <span>Subtotal</span>
+            </div>
+            {cart.map((cartItem) => (
+              <CartRow
+                key={cartItem.product.id}
+                cartItem={cartItem}
+                cart={cart}
+                setCart={setCart}
+              />
+            ))}
+            <div className="cart-totals">
+              Total:{" "}
+              <span className="sum">
+                $
+                {cart
```

Comment:
You can extract this in a separate method in a _utils.ts_ file, as you calculate the total amount in order-datails too.